src/main/java/teammates/common/util/Const.java

```diff
@@ -265,6 +265,7 @@ private Const() {
         public static final String INSTRUCTOR = URI_PREFIX + "/instructor";
         public static final String INSTRUCTOR_PRIVILEGE = URI_PREFIX + "/instructor/privilege";
         public static final String RESULT = URI_PREFIX + "/result";
+        public static final String PAGINATION_META_DATA = URI_PREFIX + "/pagination/metadata";
```

Comment:
Does this endpoint have to reside in a separate routing group? If so, what other calls could there possibly be in the same `/pagination/` routing group?